arangod/RestHandler/RestUsersHandler.cpp

```diff
@@ -295,7 +297,8 @@ static Result StoreUser(auth::UserManager* um, int mode,
         }
       }
       if (extra.isObject() && !extra.isEmptyObject()) {
-        entry.setUserData(VPackBuilder(extra));
+        VPackSlice oldExtra = u.get("extra");
```

Comment:
We might need to check here if "oldExtra" is an object. What happens if it is a string or a numeric value and gets merged then by "VPackCollection::merge"?

Reply:
Fixed in https://github.com/arangodb/arangodb/pull/16675/commits/22e2f0e6b470bc0b0d5d14ddd0e5b10a661b5b99